pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1180,13 +1180,13 @@ func (c *PodAffinityChecker) InterPodAffinityMatches(pod *v1.Pod, meta algorithm
 	return true, nil, nil
 }
 
-// podMatchesPodAffinityTerms checks if the "targetPod" matches the given "affinityKind"
+// podMatchesPodAffinityTerms checks if the "targetPod" matches the given "affinityProp"(true:affinity,false:anti-affinity)
 // of the "pod" on the given "nodeInfo".Node(). It returns four values: 1) whether
 // targetPod matches any of the affnityTerms and their topologies, 2) whethertargetPod matches
 // all the affnityTerms and their topologies, 3) whether targetPodmatches all the terms label
 // selector and namespaces (AKA term properties), 4) any error.
-func (c *PodAffinityChecker) podMatchesPodAffinityTerms(pod, targetPod *v1.Pod, nodeInfo *schedulercache.NodeInfo, affinityKind string) (bool, bool, bool, error) {
-	matchedTerms, err := getMatchingAffinityTerms(pod, targetPod, affinityKind)
+func (c *PodAffinityChecker) podMatchesPodAffinityTerms(pod, targetPod *v1.Pod, nodeInfo *schedulercache.NodeInfo, affinityProp bool) (bool, bool, bool, error) {
```

Comment:
`affinityProp` is a little confusing to me... Could you rename to sth. like `checkAffinity` or `checkAntiAffinity`? or making it a new type: `type PodAffinityKind string`.

Reply:
ok ,i think checkAffinity is a very good name